security/iam/aws/open_iam_policy.pt

```diff
@@ -0,0 +1,137 @@
+name "AWS Open IAM Policy"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for iam policies that have access to everything."
+long_description "Version 0.1"
+severity "high"
+category "Security"
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+parameter "escalate_to" do
+  type "string"
+  label "Email address to send escalation emails to"
+  default "edwin@rightscale.com"
```

Comment:
remove the default, add description, and use consistent label as other policies have.